tests/unit/kvmagent/test_mevoco_handlers.py

```
@@ -0,0 +1,836 @@
+_ = sys.modules.setdefault("plugin", MagicMock())
+_ = sys.modules.setdefault("traceable_shell", MagicMock())
+_ = sys.modules.setdefault("report", MagicMock())
+_ = sys.modules.setdefault("linux", MagicMock())
+_ = sys.modules.setdefault("pyparsing", MagicMock())
+
+try:
+    mevoco = cast(
+        _MevocoModule,
+        cast(object, importlib.import_module("kvmagent.plugins.mevoco")),
+    )
+except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)